server/models/db/school.js

```diff
@@ -0,0 +1,30 @@
+const _ = require('lodash');
+
+const { db } = require('../../lib/connection');
+
+const UserTypes = require('../../constants/userTypes.json');
+const Tables = require('../../constants/tables.json');
+
+const WorkerDetails = require('../data/Worker');
+
+// This method returns the corresponding worker ids given school ID. If the school ID is incorrect, it will return null. 
+// If either the tableName or Object are null or undefined, an error will be thrown.
+async function getWorkerIdsForSchool(schoolId) {
+    if (_.isNil(schoolId)) return null;
+    try {
+        //Query to recieve worker IDs from Service Worker School Table 
+        const workerIds = await db.any(`select worker_id from Service_Worker_School where school_id='${schoolId}';`);
```

Comment:
HI Simran,
To increase code consistency, I would recommend using the tables constant file for the table name, e.g., `Tables.serviceWorkerSchool` and add that appropriate value to the constant table json file.

Reply:
Hi Amy! 

I've resolved this comment by changing the name as described by you! 

Thanks a lot! 

Simran